frontend/src/components/Login/Login.jsx

```diff
@@ -29,6 +30,10 @@ const textContent = {
 
 function Login(props) {
   const content = textContent[props.type];
+  const [show, setShow] = useState(false);
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
```

Comment:
Do we need to assign this? isn't setShow already doing this.

Reply:
Ah, I just copied that code from Bootstrap's Modal docs, we can remove the functions if required